server/src/main/java/org/elasticsearch/action/ActionModule.java

```diff
@@ -434,6 +436,7 @@
 import org.elasticsearch.rest.action.search.RestMultiSearchAction;
 import org.elasticsearch.rest.action.search.RestSearchAction;
 import org.elasticsearch.rest.action.search.RestSearchScrollAction;
+import org.elasticsearch.rest.action.synonyms.RestPutSynonymsAction;
```

Comment:
Registered a new PUT action. Please check that the different actions are in the expected packages for server 🙏

Reply:
This package is fine, or we can also put this action into `org.elasticsearch.rest.action.admin.indices` as it is about .synonyms index